crypto/timing_safe_equal_test.ts

```diff
@@ -62,11 +62,59 @@ Deno.test({
 });
 
 Deno.test({
-  name: "[timingSafeEqual] - Uint8Array comparison - not equal",
+  name: "[timingSafeEqual] - Uint8Array comparison #2 - not equal",
   fn() {
     const encoder = new TextEncoder();
     const a = encoder.encode("hello deno");
     const b = encoder.encode("hello Deno");
     assert(!timingSafeEqual(a, b));
   },
 });
+
+Deno.test({
+  name: "[timingSafeEqual] - Uint8Array w. different buffer sizes (a > b)",
+  fn() {
+    const a = new SharedArrayBuffer(4);
```

Comment:
minor point, but do we need to use SharedArrayBuffer here? How about ArrayBuffer?

Reply:
`ArrayBuffer` is a transferable object, so we won't be able to create `Uint8Array` after manipulating it with the `DataView`. And we do want to do that, to make `Uint8Array` a specific slice of the buffer for the purpose of these specific tests.